playwright.config.ts

```diff
@@ -5,7 +5,9 @@ import * as dotenv from 'dotenv';
  * https://github.com/motdotla/dotenv
  */
 
-dotenv.config({ path: ['.env.playwright.local', '.env.playwright'] });
+dotenv.config({
+  path: ['.env.playwright.production', '.env.playwright.local', '.env.playwright'],
```

Comment:
for what do we need .env.playwright.production if we already have .env.playwright.local ? if we still need it then should we update readme ?